src/transformers/models/altclip/processing_altclip.py

```diff
@@ -16,10 +16,33 @@
 Image/Text processor class for AltCLIP
 """
 
-import warnings
+from typing import List, Union
 
-from ...processing_utils import ProcessorMixin
-from ...tokenization_utils_base import BatchEncoding
+
+try:
+    from typing import Unpack
+except ImportError:
+    from typing_extensions import Unpack
+
+from ...image_utils import ImageInput
+from ...processing_utils import (
+    ProcessingKwargs,
+    ProcessorMixin,
+)
+from ...tokenization_utils_base import BatchEncoding, PreTokenizedInput, TextInput
+from ...utils import is_torch_available, is_vision_available
+from ...utils.deprecation import deprecate_kwarg
+
+
+# TODO (@molbap) This is a bother, forward references from TypedDict are resolved and need this to work
```

Comment:
Do we have an issue open for this, so that we don't forget?

Reply:
ah, this is outdated - with the new import structure of Unpack this issue is not there anymore!